components/vf-link-list/vf-link-list--easy.njk

```diff
@@ -0,0 +1,25 @@
+<div class="vf-links {%- if component_modifier %} {{ component_modifier }}{% endif %}">
+  <h3 class="vf-links__heading">{{ title }}</h3>
+  <ul class="vf-links__list {%- if list_modifier %} vf-links__list--{{ list_modifier}}{% endif %} | vf-list">
+    {% for item in list %}
+    <li class="vf-list__item">
+      <a class="vf-list__link" href="{{ item.link_list_href }}">
+        {% if item.image %}
+          {% if item.image == "blank" %}
+            <span><!-- no image --></span>
+            {% else %}
+            {{ item.image }}
+          {% endif %}
+        {% endif %}
+        {{ item.text }}
+
+        <svg class="vf-icon vf-icon__arrown--down | vf-list__icon" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><title>arrow-button-down</title><path d="M.249,7.207,11.233,19.678h0a1.066,1.066,0,0,0,1.539,0L23.751,7.207a.987.987,0,0,0-.107-1.414l-1.85-1.557a1.028,1.028,0,0,0-1.438.111L12.191,13.8a.25.25,0,0,1-.379,0L3.644,4.346A1.021,1.021,0,0,0,2.948,4a1,1,0,0,0-.741.238L.356,5.793A.988.988,0,0,0,0,6.478.978.978,0,0,0,.249,7.207Z"/></svg>
```

Comment:
The entropy of inline SVGs for the HTML template examples still makes me worry, but I get that's where we're at for the now.

Reply:
> The 'on this page' list needs an SVG icon -- We need to work out if inline is the best approach, or if we should be creating a sprite. We'd need to be able to style the SVG w/ CSS (at least)

Missed that

Reply:
But, yes, very much so :)

Reply:
I have in the past used `use` with the SVG pulled inline at the top of the page with a `display: none` attached. Which worked. Following several conversation on SVGs over the years it seems that 'best practice' is to inline (even with the issue of maintainability)

At least we have a confirmed pool of icons to go from - so the chance of their being a drastic change is slightly reduced.

Reply:
Incase it gets lost in the notifications: https://github.com/visual-framework/vf-core/issues/751#issuecomment-594417259

Reply:
With the comment in #751 I'll resolve this conversation